libs/editor/egui_editor/src/input/mutation.rs

```diff
@@ -327,17 +328,13 @@ pub fn calc(
 
             // always iterate forwards when renumbering because numbers are based on prior numbers for both indent
             // and deindent operations
-            processed_galleys.clear();
```

Comment:
second, because we cleared this, only when you renumbered a galley would it register as a change and skip inserting the tab, so when you indented a bullet it would still insert a tab...